spec/defines/repo_spec.rb

```diff
@@ -225,12 +225,22 @@
     it { should contain_file('rspec.repo').with_content(/\[rspec\]\nfailovermethod=priority/) }
   end
 
+  context 'with sslcacert set to valid string </path/to/cert>' do
+    let(:params) { mandatory_params.merge({ :sslcacert => '/path/to/cert' }) }
+    it { should contain_file('rspec.repo').with_content(/\[rspec\][\s\S]*sslcacert=\/path\/to\/cert/) }
+  end
+
+  context 'with sslcacert undefined' do
+    let(:params) { mandatory_params.merge({ :sslcacert => :undef }) }
+    it { should contain_file('rspec.repo').without_content(/\[rspec\][\s\S]*sslcacert=/) }
+  end
+
   describe 'variable type and content validations' do
     let(:mandatory_params) { mandatory_params }
 
     validations = {
       'absolute_path' => {
-        :name    => %w(gpgkey_local_path yum_repos_d_path),
+        :name    => %w(gpgkey_local_path yum_repos_d_path sslcacert),
```

Comment:
You'll notice that nil (:undef) is specified as invalid while for this param it is OK.

It should have a new category like `'absolute_path_and_undef'`